source/js/buyers-guide/bg-main.js

```diff
@@ -8,6 +8,8 @@ import navNewsletter from "../nav-newsletter.js";
 import CreepVote from "./components/creep-vote/creep-vote.jsx";
 import Creepometer from "./components/creepometer/creepometer.jsx";
 import JoinUs from "../components/join/join.jsx";
+import { LocalizationProvider } from "@fluent/react/compat";
```

Comment:
Depending on your build setup, you might not need the `compat` build. It might be a good idea to try importing from `@fluent/react`.

Reply:
`"@fluent/react"` seem to work just fine, I thought this was needed to support older browsers

Reply:
It all depends on your build setup and compatibility story. If you don't transpile `node_modules`, you might want to keep using `compat` here. Also, keep in mind that even `compat` targets browsers capable of supporting ES2017.